projects/cds/src/merchandising/adapters/strategy/cds-merchandising-strategy.adapter.ts

```diff
@@ -19,13 +20,18 @@ export class CdsMerchandisingStrategyAdapter
   ) {}
 
   loadProductsForStrategy(
-    strategyId: string
+    strategyId: string,
+    strategyRequest?: StrategyRequest
```

Comment:
Can we update the test to check if this `strategyRequest` is delegated to the `cdsEndpointsService` (if passed)?

Reply:
I've updated the test to include this